layer.md

```diff
@@ -1,63 +1,129 @@
-# Creating an Image Filesystem Changeset
+# Filesystem Layer Changeset
 
-An example of creating an Image Filesystem Changeset follows.
+A layer changeset is the distributable difference between filesystems.
+A layer changeset is not directly extracted, but applied to a filesystem.
 
-An image root filesystem is first created as an empty directory.
-Here is the initial empty directory structure for a changeset using the randomly-generated directory name `c3167915dc9d` ([actual layer DiffIDs are generated based on the content](#id_desc)).
+## Distributable Format
+
+Layer Changesets for the [mediatype](./media-types.md) `application/vnd.oci.image.layer.tar+gzip` MUST be packaged in [tar archive][tar-archive].
+Layer Changesets for the [mediatype](./media-types.md) `application/vnd.oci.image.layer.tar+gzip` MUST NOT include duplicate entries for file paths in the resulting [tar archive][tar-archive].
+
+## Change Types
+
+Types of changes that can occur in a changeset are:
+
+* Additions
+* Modifications
+* Removals
+
+Additions and Modifications are reflected the same in the changeset tar archive.
```

Comment:
On Wed, Sep 07, 2016 at 06:31:22AM -0700, Vincent Batts wrote:
> > +Additions and Modifications are reflected the same in the changeset tar archive.
>
> I'm not inclined to just stick with layer. As Layer is generic
> enough to refer to the on disk representation, where changeset is
> the specific mechanics being described.

If you don't like “Layer tar archive”, how about following the pattern
you use in the rest of this file and just saying “tar archive”?

And if you see an important distinction between “layer (changeset)”
and “changeset”, you should probably file a PR rolling back #243 and
explaining the distinction.